images/aws-kms-encryption-provider.yml

```diff
@@ -0,0 +1,30 @@
+content:
+  source:
+    dockerfile: Dockerfile.openshift
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/aws-encryption-provider.git
+      web: https://github.com/openshift/aws-encryption-provider
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-build-root
+        auto_label:
+        - approved
+        - lgtm
```

Comment:
We do not add this normally. Only storage components historically have this. Unless the image owner has specifically requested for this, it can be removed.